jobs/JGFS_ATMOS_AWIPS_20KM_1P0DEG

```diff
@@ -27,16 +25,15 @@ export COMPONENT="atmos"
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/${COMPONENT}}
-export COMOUTwmo=${COMOUTwmo:-${COMOUT}/wmo}
-
 export SENDDBN=${SENDDBN:-NO}
 export SENDAWIP=${SENDAWIP:-NO}
 export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 
-if [ ${SENDCOM} = YES ] ; then
-  mkdir -m 775 -p ${COMOUT} ${COMOUTwmo}
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_WMO
+RES="0p25" YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_GRIB_0p25:COM_ATMOS_GRIB_TMPL
```

Comment:
the file says 1P0DEG, is 0p25 correct here?

Reply:
0p25 is what [scripts/exgfs_atmos_awips_20km_1p0deg.sh](https://github.com/NOAA-EMC/global-workflow/pull/1421/files/460755da615af92a071648f2551ce5be499d0379#diff-76b77a2849787a7c2a3cf3b8239baf87a26aa2e78b3d2f39dfdb2a4a8cca9fa8) actually looks for. I'm sure the scripts just never got renamed. These scripts have not had as much TLC as they should have.